CONTRIBUTING.md

```diff
@@ -3,25 +3,28 @@
 ## Getting started <!-- omit in toc -->
 
 Before you begin:
-- This site is powered by Node.js. Check to see if you're on the [version of node we support](contributing/development.md).
 - Have you read the [code of conduct](CODE_OF_CONDUCT.md)?
 - Check out the [existing issues](https://github.com/github/docs/issues) & see if we [accept contributions](#types-of-contributions-memo) for your type of issue.
 
-### Use the 'make a contribution' button
+### Don't see your issue? Open one
 
-<img src="./assets/images/contribution_cta.png" width="400">
+If you spot something new, open an issue using a [template](https://github.com/github/docs/issues/new/choose). We'll use the issue to have a conversation about the problem you want to fix.
 
-Navigating a new codebase can be challenging, so we're making that a little easier. As you're using docs.github.com, you may come across an article that you want to make an update to. You can click on the **make a contribution** button right on that article, which will take you to the file in this repo where you'll make your changes.
+**Note:** We cannot accept contributions to the [REST API reference documentation](https://docs.github.com/en/rest/reference). If you spot an inaccuracy in the REST API reference documentation, open an issue in the [github/rest-api-description](https://github.com/github/rest-api-description/issues/new?template=schema-inaccuracy.md) repository.
+
+## Ready to make a change? 
 
 Before you make your changes, check to see if an [issue exists](https://github.com/github/docs/issues/) already for the change you want to make.
 
-### Don't see your issue? Open one
+### Use the 'make a contribution' button
 
-If you spot something new, open an issue using a [template](https://github.com/github/docs/issues/new/choose). We'll use the issue to have a conversation about the problem you want to fix.
+<img src="./assets/images/contribution_cta.png" width="400">
 
-**Note:** We cannot accept contributions to the [REST API reference documentation](https://docs.github.com/en/rest/reference). If you spot an inaccuracy in the REST API reference documentation, open an issue in the [github/rest-api-description](https://github.com/github/rest-api-description/issues/new?template=schema-inaccuracy.md) repository.
+Navigating a new codebase can be challenging, so we're making that a little easier. As you're using docs.github.com, you may come across an article that you want to make an update to. You can click on the **make a contribution** button right on that article, which will take you to the file in this repo where you'll make your changes.
+
+### Fork the repo
 
-### Ready to make a change? Fork the repo
+This site is powered by Node.js. Check to see if you're on the [version of node we support](contributing/development.md).
```

Comment:
Moving this here since this info is most relevant to people forking the repository to set up a local version of the site.